src/scales/quantitativeScale.ts

```diff
@@ -42,6 +63,40 @@ module Plottable {
       return super.domain(values); // need to override type sig to enable method chaining:/
     }
 
+    /**
+     * Gets the minimum domain value when autoDomain()-ing. 
+     */
+    public autoMin(): D;
+    /**
+     * Sets the minimum domain value when autoDomain()-ing.
+     */
+    public autoMin(value: D): QuantitativeScale<D>;
+    public autoMin(value?: D): any {
+      if (value === undefined) {
+        return this._autoMin;
+      }
+      this._autoMin = value;
+      this._autoDomainIfAutomaticMode();
```

Comment:
`_autoDomainIfAutomaticMode()` is getting renamed, but I think "`autoDomain()`" is a good name.